sword2/connection.py

```diff
@@ -300,7 +300,7 @@ def _handle_error_response(self, resp, content):
             conn_l.error("You are unauthorised (401) to access this document on the server. Check your username/password credentials and your 'On Behalf Of'")
             return self._return_error_or_exception(NotAuthorised, resp, content)
         elif resp['status'] == 403:
-            conn_l.error("You are Forbidden (401) to POST to '%s'. Check your username/password credentials and your 'On Behalf Of'")
+            conn_l.error("You are Forbidden (403) to POST to '%s'. Check your username/password credentials and your 'On Behalf Of'")
```

Comment:
automatically converted by `1to3` I guess :laughing:

Reply:
I changed it manually :) Error message should also be 403 when response status is 403 right?